packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -61,60 +79,64 @@
 .label-value {
   inset-block-end: calc(100% + 0.5em);
   font-weight: var(--calcite-font-weight-bold);
-  color: var(--calcite-color-text-1);
+  color: var(--calcite-meter-value-text-color, var(--calcite-color-text-1));
 }
 
 .label-range {
   @apply text-color-3;
+  color: var(--calcite-meter-range-text-color, var(--calcite-color-text-3));
   inset-block-start: calc(100% + 0.5em);
 }
 
 .unit-label {
-  @apply text-color-3 font-medium;
+  color: var(--calcite-meter-unit-text-color, var(--calcite-color-text-3));
+  font-weight: var(--calcite-font-weight-medium);
 }
 
 .label-value .unit-label {
-  @apply text-color-2 font-bold;
+  color: var(--calcite-meter-unit-text-color, var(--calcite-color-text-2));
+  font-weight: var(--calcite-font-weight-bold);
 }
 
 .fill {
-  @apply block absolute duration-150 ease-in-out bg-brand z-default;
-  inset-inline-start: var(--calcite-meter-space-internal);
-  inset-block: var(--calcite-meter-space-internal);
-  border-radius: var(--calcite-meter-height-internal);
-  max-inline-size: calc(100% - (var(--calcite-meter-space-internal) * 2));
-  min-inline-size: calc(var(--calcite-meter-height-internal) - (var(--calcite-meter-space-internal) * 2));
+  @apply block absolute duration-150 ease-in-out z-default;
+  inset-inline-start: var(--calcite-internal-meter-space);
+  inset-block: var(--calcite-internal-meter-space);
+  border-radius: var(--calcite-meter-corner-radius, 9999px);
```

Comment:
This `9999px` needs to made into a token - in TW this is `rounded-full`

Reply:
@alisonailea should we leave this for now? Add a comment?

Reply:
I've made a note to add this token but we can leave it as is for now